src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        public static bool Is64BitOS()
+        {
+            return Architecture.BIT64 == ArchitectureManager.OSArchitecture;
+        }
+
+        public static bool Is64BitProcess()
+        {
+            return Architecture.BIT64 == ArchitectureManager.ProcessArchitecture;
+        }
+
+        public static bool IsArmProcessor()
```

Comment:
API Comment: This implies that there's "ARM" and "Other" in someone's decision-making process.  Consider future evolution: X86, ARM, SPARC(64), PPC, etc.  Making simple "Is" feels like it's setting a lot of applications up for bad behavior.